app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html

```diff
@@ -74,8 +74,8 @@
                     }
                     @if(event.details.`type` == EventType.CASE_STATUS_CHANGE) {
                         @progressive_disclosure(
-                            id = s"activity-events-row-${event.id}",
-                            summary = s"Status changed to ${event.details.asInstanceOf[CaseStatusChange].to.toString.toLowerCase}",
+                            id="case-note-comment",
```

Comment:
This will definately break some view tests

Reply:
Also will give loads of console errors when theres multiple activities on one case (you cant have more than one element with the same ID in the entire document)